viewing_party/party.py

```diff
@@ -1,23 +1,134 @@
 # ------------- WAVE 1 --------------------
 
+import math
+
+
 def create_movie(title, genre, rating):
-    pass
+    movie = {}
+    if title and genre and rating:
+        movie["title"] = title
+        movie["genre"] = genre
+        movie["rating"] = rating
+        return movie    
+    else:
+        return None    
+
+def add_to_watched(user_data, movie):
+    if not "watched" in user_data:
+        user_data["watched"] = []
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    if not "watchlist" in user_data:
+        user_data["watchlist"] = []
+    user_data["watchlist"].append(movie)
+    return user_data
+    
+def watch_movie(user_data, title):
+    for ele in user_data["watchlist"]:
+        if title == ele["title"]:
+            user_data["watched"].append(ele)
+            user_data["watchlist"].remove(ele)
+            return user_data

```

Comment:
this works! we can also let the final return statement take care of it and remove this one